### PR TITLE
Fix GitHub actions running checks

### PR DESCRIPTION
**Pull request by JenySadadia**
Signed-off-by: Jeny Sadadia <jeny.sadadia@gmail.com>

### CODE REVIEW

**Comment by JenySadadia** [comment]
Tested on 'fix-python-version' branch. Github actions check completed successfully.
https://github.com/JenySadadia/kernelci-core/actions/runs/1626830604

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -19,21 +19,23 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        python-version: [3.7, 3.8, 3.9]
+        python-version: ['3.10'] # Only supported one at the moment
```
Comment (gctucker) [inline .github/workflows/main.yml:22]: Actually that was true for kernelci-api, but I believe kernelci-core might work with earlier versions.  So it's worth trying with 3.7 and 3.10 to see what happens.

**Review by gctucker — COMMENTED** [review]
LGTM - just maybe one thing to tweak to see if earlier Python versions work as well.

File: .github/workflows/main.yml (hunk 1 above)
Comment (JenySadadia) [reply .github/workflows/main.yml]: I have checked with python version 3.7, 3.8, 3.9, and 3.10
https://github.com/kernelci/kernelci-core/actions/runs/1626891867
It is working with all the versions.

**Review by gctucker — APPROVED** [review]

**Pull request merged**